src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -321,15 +322,16 @@ public void close(Collection<TopicPartition> partitions) {
               partitionsToChannel.get(partitionChannelKey);
           // Check for null since it's possible that the something goes wrong even before the
           // channels are created
-          if (topicPartitionChannel != null) {
-            topicPartitionChannel.closeChannel();
-          }
           LOGGER.info(
-              "Closing partitionChannel:{}, partition:{}, topic:{}",
+              "Removing partitionChannel:{}, partition:{}, topic:{} from map(partitionsToChannel)",
```

Comment:
Are we relying on the channel cleaner to close the channel then? Also it is slightly concerning that we can make this behavior change and not break any of our existing tests

Reply:
Channel cleaner doesnt close the channel. Channel cleaner cleans up inactive channels. (Those who dont have any commit activity in last 30 days)

Valid concern regarding tests not failing, I will take a look.

Reply:
On the other hand, its kind of safe to remove it since none of our tests are failing. 
I checked our existing test and it kind of makes sense to remove it since there was no behavior change:
Checkout `testRebalanceOpenCloseIngestion` and `testChannelCloseIngestion`

It would have failed if I had checked for channel .isClosed() but that wasnt present in any test.

Reply:
Yeah there are pros and cons to having our tests be fine grained enough to catch behavior like this. 

I am confused about the channel cleaner - does it close the channel as part of cleaning up inactive channels? I'm mostly concerned that we'll be leaving behind a lot of orphaned channels, especially as we currently have this bug with cleaning up old inactive channels 

https://snowflakecomputing.atlassian.net/browse/SNOW-800927

Reply:
`StreamingIngestChannelCleaner` is the class if you want to go through the logic. 
But essentially, it removes the channels from FDB. If you are seeing old channels, it might be worth investigating why this is happening. - might be a bug somewhere. Also note that it scans the lastCommitTime, not createdOn. 


> I am confused about the channel cleaner - does it close the channel as part of cleaning up inactive channels?
There is no callback but lets say from client side if you still have a channel instance and channel gets wiped off from server side, you will start getting exception. (Note that, you will have to reopen the channel)

> I'm mostly concerned that we'll be leaving behind a lot of orphaned channels
Umm, we should investigate it why is this the case because this seems to be a bug then.

Reply:
Sounds good! thanks for the explanation. Its something on my backlog, just havent had time to look into it yet

Reply:
No problem, this is something I have looked on, going to discuss this on monday meeting, i can take a look at the jira and gs logs!